### DISCUSSION TITLE
网络

### DISCUSSION THREAD

**Discussion by QVQ-QWQ** [share121/inter-knot#2279] [其他]
![7438507e90e166fd629b6e87c3255b9](https://github.com/user-attachments/assets/666682d0-83ee-4e8a-ae40-2b2c5ff98638)

有的时候进不去网站。
还有一件事！电脑前坐的腰疼，希望出个移动端教程

**Comment by Maixiaofanqie** [comment]
似乎现在只能走电脑，毕竟移动端的edge似乎没法用油猴，进不去网站可以挂个加速器，一般加速器都有加速Github的吧<img src="https://bbs-static.miyoushe.com/static/2023/11/16/b7b6eb7ebdc9f63ba6365e7e8d6b5fd7_5912345410510420967.png" alt="安比-啊···" width="75" >

**Reply by KUYUE404** [reply]
移动端可以用via浏览器，可以直接下载插件，不用油猴

**Reply by zhuongxt** [reply]
还可以用狐猴浏览器

**Comment by QVQ-QWQ** [comment]
还有一件事！我的搜索为什么用不了

**Reply by share121** [reply]
因为我还没开发

**Reply by anything12121** [reply]
捉！(希望作者快点开发)

**Comment by biliqide** [comment]
可以用via，我移动端用的就是，我现在就是用手机回复你的🤓

**Reply by QVQ-QWQ** [reply]
> 可以用via，我移动端用的就是，我现在就是用手机回复你的 🤓

多谢

**Comment by QVQ-QWQ** [comment]
还有一件事！今天抽卡又歪了
![631c89d1-191b-4c6b-be97-33905094c5fd](https://github.com/user-attachments/assets/f472c346-72ad-4e1a-a443-4c2f0cf6e69c)

**Reply by dolokong** [reply]
楼主怎么在评论区发的表情包

**Reply by QVQ-QWQ** [reply]
你可以在绳网里翻一下，有人发布的插件

**Comment by dolokong** [comment]
![IMG_20240813_161907](https://github.com/user-attachments/assets/1887e941-71b7-4174-9393-57e21c9b3ada)

**Comment by hk0852** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by hk0852** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by QVQ-QWQ** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/729e37ee656d8dba288b4f0959602577_6148928601806065672.png" alt="赛诺-查岗" width="75" >

**Comment by QVQ-QWQ** [comment]
感觉用手机登比电脑快多了
<img src="https://upload-bbs.mihoyo.com/upload/2022/11/24/5b2ddd1000630c0f9542dea01dd715aa_4420398458026402084.png" alt="纳西妲-开心" width="75" >

**Comment by JobNumber-9527** [comment]
问一下，头像怎么换啊，他一直说“我们处理不了那张图片”
